CHANGELOG.md

```diff
@@ -6,6 +6,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.4] - 2022-08-22
+
+> (Some codename goes here)
```

Comment:
(This PR is a draft as I can't come up with any codename. Suggestion wanted 🙏)

Reply:
"I've been through the desert on a horse with no name" ❓ 😂  (https://genius.com/America-a-horse-with-no-name-lyrics)

Reply:
Thanks, but it seems rumored that the lyrics has some unexpected (?) meaning? (Accidentally?)
https://en.wikipedia.org/wiki/A_Horse_with_No_Name#Reception

Reply:
TIL. sight.. so a song with `horse` in the title is now "bad"? 🙄. Thanks for finding that (not sure if a single city in the USA banning it in 1972 is reason to skip it 😅). 

Don't have good suggestion from the top of my head for something else 🤔  Could post some lyrics from Marillion, but they're not all "peachy" 😅 

_"On the outskirts of nowhere. On the ring road to somewhere. On the verge of indecision. I'll always take the roundabout way"_
from https://www.marillion.com/music/lyric.htm?id=31

_"When footlights dim in reverence to prescient passion forewarned, my audience leaves the stage"_
https://www.marillion.com/music/lyric.htm?id=25

Reply:
According to https://www.countrythangdaily.com/america-horse-no-name/ , the `horse` seems fine 😅 but I guess it is still safer to avoid citing copyrighted lyrics?